fiftyone/server/routes/media_reference.py

```
@@ -0,0 +1,440 @@
+        _raise_resolution_error(422, "malformed-media-source", exc)
+    except UnsupportedMediaReferenceOperation as exc:
+        _raise_resolution_error(415, "unsupported-media-reference", exc)
+    except MediaReferenceError as exc:
+        _raise_resolution_error(422, "malformed-media-reference", exc)
+    except OSError as exc:
+        _raise_resolution_error(409, "stale-media-reference", exc)
+
+    return dataset, sample, manifest
+
+
+def _raise_resolution_error(status_code, kind, error):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_raise_resolution_error`

Add return type annotation: `NoReturn`

(ANN202)